lib/components/circular_button.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:FoodForGood/constants.dart';
+import 'package:flutter/material.dart';
+
+class CircularButton extends StatelessWidget {
+  final Color splashColour;
+  final Color colour;
+  final String title;
+  final Function pressed;
+  final height;
+  final width;
+
+  CircularButton({
+    this.colour,
+    this.title,
+    this.pressed,
+    this.splashColour = Colors.deepOrange,
+    this.height = 150.0,
+    this.width = 150.0
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return ClipOval(
+      child: Material(
+        color: kPrimaryColor, // button color
+        child: InkWell(
+          splashColor: splashColour,
```

Comment:
Use `this.` here and elsewhere for all member variables.

Reply:
Done